src/main/java/TaskManager.java

```diff
@@ -0,0 +1,289 @@
+                                for (int n = taskNo - 1; n < i; n++) {
+                                    taskList[n] = taskList[n + 1];
+                                }
+                            }
+                            i--;
+                        }
+                    }
+                }
+            } else {
+                errorUnknownCommandMessage();
+            }
+        } else {
+            byeMessage();
+            try {
+                for (int k = 0; k < i; k++) {
+                    String done;
+                    if (taskList[k].getStatusIcon() == "X") {
+                        done = "1|";
+                    } else {
+                        done = "0|";
+                    }
+                    if (taskList[k].getTask() == "T") {
+                        String command = done + k + 1 + "|todo " + taskList[k].getDescription();
+                        if (k == 0) {
+                            writeToFile(path, command);
+                        } else {
+                            appendToFile(path, command);
+                        }
+                    } else if (taskList[k].getTask() == "D") {
+                        String command = done + k + 1+ "|deadline " + taskList[k].getDescription() + "/by " + taskList[k].getDateNum();
+                        if (k == 0) {
+                            writeToFile(path, command);
+                        } else {
+                            appendToFile(path, command);
+                        }
+                    } else if (taskList[k].getTask() == "E") {
+                        String command = done + k + 1+ "|event " + taskList[k].getDescription() + "/at " + taskList[k].getDateNum();
+                        if (k == 0) {
+                            writeToFile(path, command);
+                        } else {
+                            appendToFile(path, command);
+                        }
+                    }
+                }
+            } catch (Exception e) {
+                System.out.println("Unable to write to file");
+            }
+        }
+
+    }
+
+    public void listAll() {
+        System.out.println("____________________________________________________________\n" +
+                "Here are the tasks in your list: ");
+        for (int j = 0; j < i; j++) {
+            if (taskList[j] instanceof ToDo) {
+                System.out.println(j + 1 + ". [" + taskList[j].getTask() + "]" +
+                        "[" + taskList[j].getStatusIcon() + "] " + taskList[j].getDescription());
+            } else {
+                System.out.println(j + 1 + ". [" + taskList[j].getTask() + "]" +
+                        "[" + taskList[j].getStatusIcon() + "] " + taskList[j].getDescription()
+                        + taskList[j].getDate());
+            }
+        }
+        System.out.println("____________________________________________________________");
+    }
+
+    public void toDoAddedMessage() {
```

Comment:
Would it be better to use an action to name this, like `printToDoAddedMessage` (similar to the other printing methods too)?